app/javascript/components/pages/Home.js

```diff
@@ -47,22 +48,22 @@ class Home extends Component {
             <UncontrolledCarousel
                 items={[
                     {
-                    altText: 'Slide 1',
-                    caption: 'Slide 1',
+                    // altText: 'Slide 1',
+                    // caption: 'Slide 1',
```

Comment:
Is this needed?  (This appears multiple times on this page, so please address this for all of those).  Please remove or uncomment